src/Modules/AuthModule/components/VerifyAccount/VerifyAccount.tsx

```diff
@@ -1,6 +1,81 @@
+import React, { useContext } from 'react';
+import { useForm } from 'react-hook-form';
+import img1 from "../../../../assets/images/PMS 3.png";
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { AuthContext } from '../../../../context/AuthContext';
 
 export default function VerifyAccount() {
-  return (
-    <div>VerifyAccount</div>
+  let {baseUrl}= useContext(AuthContext)
+
+    const { handleSubmit, register,formState: { errors } } = useForm();
+  const Navigate=  useNavigate()
+
+    async function onSubmit (data:any) {
+        try {
+            const res = await axios.put(`${baseUrl}/Users/verify`, data);
+            console.log(res);
+            toast.success(res?.data?.message)
+            Navigate('/login')
+        } catch (error) {
+            console.log(error);
+            toast.success(error?.res?.data?.message || "there's an error" )
+
+        }
+        
+    }
+
+    return (
+     <>
+        <div className="Auth-container">
+            <div className="container-fluid ">
+                <div className="row justify-content-center align-items-center">
+                    <div className="col-md-6">
+                        <div className="image text-center">
+                            <img src={img1} alt="" />
+                        </div>
+                        <form onSubmit={handleSubmit(onSubmit)} className='p-5 rounded-4 my-5'>
+                            <div className="text text-white ms-3">
+                                <p className='text-Auth'>Welcome to PMS</p>
+                                <h2>Verify Account</h2>
+                            </div>
+                            <div className="row">
+      
+                                <div className="col-md-12 ">
+                                    <div className="input m-3">
+                                        <label className='my-1'>Email</label>
+                                        <input type="email" className='form-control p-0' placeholder='Enter Your E-mail' {...register('email', { required: 'Email is required', pattern: { value: /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/, message: 'Email is invalid' } })} />
```

Comment:
It seems here that we are not using formatter like prettier. I encourage you as a team to setup prettier to have the same code format, this will make the code easier to read. You can make it format on save, so when you save your files, it will automatically be formatted.